test/expression/micro/parser_test.dart

```diff
@@ -41,6 +41,26 @@ void main() {
     );
   });
 
+  test('should parse a let with a full Dart expression', () {
+    expect(
+      parse('ngFor', 'let x of items.where(filter)'),
```

Comment:
Ah, but I think _here_ is where I got you in terms of wanting to use the dart tokenizer!

```
  let x of [";", ",", "."]
```

I think that will fail